ntbs-service/Startup.cs

```diff
@@ -350,14 +350,16 @@ private void ConfigureHangfire(IApplicationBuilder app)
             app.UseHangfireDashboard("/hangfire", dashboardOptions);
             app.UseHangfireServer(new BackgroundJobServerOptions { WorkerCount = 1 });
             GlobalJobFilters.Filters.Add(new AutomaticRetryAttribute { Attempts = 0 });
-            
+
             if (!Env.IsDevelopment())	
             {
                 // Most of the time we don't care about recurring jobs in dev mode.
-                // Having this exclusion is also useful when connecting to non-dev databases for debugging.
-                // as jobs scheduled from (windows) dev machines won't run on linux due to different timezone formats
-                // Comment out this check to work with jobs locally.
-                HangfireJobScheduler.ScheduleRecurringJobs();
+                // Having this exclusion is also useful when connecting to non-dev databases for debugging
+                // as jobs scheduled from (windows) dev machines won't run on linux due to different timezone formats.
+                // When running locally confirm jobs to be ran are enabled in configuration.
+                var scheduledJobConfig = new ScheduledJobConfig();
+                Configuration.GetSection("ScheduledJobConfig").Bind(scheduledJobConfig);
+                HangfireJobScheduler.ScheduleRecurringJobs(scheduledJobConfig);
```

Comment:
This binding seems correct - I'd be surprised if the DI that does it magically for us outside of startup did something that didn't boil down to this! :)